### ISSUE TITLE
keptn configure monitoring prometheus responds with configmaps "alertmanager-templates" already exists

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn-contrib/prometheus-service#197]
I already had prometheus installed for another project, but when I ran `keptn configure monitoring prometheus --project=sockshop --service=carts`, I got this error:

```
$ keptn configure monitoring prometheus --project=sockshop --service=carts
ID of Keptn context: 4abe6c1f-539f-4c92-a32b-63682b686b0b
configmaps "alertmanager-templates" already exists
```

# Action Items

- [ ] Investigate why this error occurs and if it causes any problems
- [ ] If it causes problems, fix it

**Comment by tibineacsu95** [comment]
Hello @christian-kreuzberger-dtx we are running into the same issue when trying to configure the prometheus service to use our prometheus instance deployed via the kube-prometheus project stack. Did you manage to find any way to bypass this error? Sorry for not posting anything related to the solution, but I just wanted to get some insights from someone who seems to have worked a lot longer with keptn than me.

**Comment by braghettos** [comment]
@christian-kreuzberger-dtx do you have an estimation on when this issue will be addressed? I'm really looking fot the 0.7.2 version of prometheus-service and this issue looks like a blocker.

**Comment by christian-kreuzberger-dtx** [comment]
Hi,

unfortunately I cannot provide an estimate just yet. Haven't had time to investigate this issue.

**Comment by christian-kreuzberger-dtx** [comment]
I have just created a pre-release without a fix for #197:

https://github.com/keptn-contrib/prometheus-service/releases/tag/0.7.2-next.0

You can install it using
```
helm install -n keptn prometheus-service https://github.com/keptn-contrib/prometheus-service/releases/download/0.7.2-next.0/prometheus-service-0.7.2-next.0.tgz
```
